pkgs/tools/system/lshw/default.nix

```diff
@@ -1,27 +1,33 @@
-{ stdenv, fetchurl
-, withGUI? false, gtk? null, pkgconfig? null, sqlite? null  # compile GUI
- }:
-stdenv.mkDerivation rec {
+{ stdenv, lib, fetchurl, fetchpatch
+, withGUI ? false, gtk2, pkgconfig, sqlite # compile GUI
```

Comment:
@joachifm yes, I'm mostly thinking of calling expressions without `callPackage` or outside of `all-packages.nix`.

Also, if I remember correctly, recently there was a difference in package sets between platforms. (There was a patch to fix that for Darwin. something related to Xorg?)